apps/dictionary/tokens/whitelabel/components/o3-tooltip.json

```
@@ -8,6 +8,16 @@
       "text-color": {
         "value": "{o3.color.palette.black-90}",
         "type": "color"
+      },
+      "icons": {
```

I'd propose singular `icon` since that follows our existing naming convention, but I don't feel strongly enough not to approve. 
```suggestion
      "icon": {
```